database-mongodb/index.js

```diff
@@ -4,10 +4,31 @@ module.exports.mongoose = mongoose;
 
 mongoose.Promise = require('bluebird');
 
-mongoose.connect('mongodb://localhost/airbrb-loc', {
+// const {
+//   // MONGO_USERNAME,
+//   // MONGO_PASSWORD,
+//   // MONGO_HOSTNAME,
+//   // MONGO_PORT,
+//   MONGO_DB
+// } = process.env;
+
+// const url = `mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_HOSTNAME}:${MONGO_PORT}/${MONGO_DB}?authSource=admin`;
+// const url = 'mongodb://mongo:27017/airbnb-location?authSource=admin';
+const url = 'mongodb://mongo:27017/airbnb-location;
+
+const options = {
   useNewUrlParser: true,
   useUnifiedTopology: true,
   useCreateIndex: true
+};
+
+//for docker
+mongoose.connect(url, options)
+.then( function() {
+  console.log('MongoDB is connected');
+})
+.catch( function(err) {
+  console.log(err);
 });
 
```

Comment:
I'm going to try this myself, but for running the app in development without containers, I think you can set up fallback hosts so you don't have to comment in/out connects for your database.